addons/docs/jest-transform-mdx.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
       import { mdx } from '@mdx-js/react'
       ${mdx.sync(src, { compilers, filepath: filename })}
     `;
-    return getNextTransformer(filename, config).transformSource(filename, result, instrument);
+    
+    const jsFileName = `${filename}.js`;
```

Comment:
wait this might not work

Reply:
nvm, it does. the file will be index.stories.mdx.js though which i think is fine